css/ol.css

```diff
@@ -37,13 +37,18 @@
 .ol-viewport, .ol-unselectable {
   -webkit-touch-callout: none;
   -webkit-user-select: none;
-  -khtml-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   -webkit-tap-highlight-color: rgba(0,0,0,0);
 }
-
+.ol-selectable {
+  -webkit-touch-callout: default;
+  -webkit-user-select: auto;
+  -moz-user-select: auto;
+  -ms-user-select: auto;
+  user-select: auto;
```

Comment:
@fredj from my testing while looking into [this question on SO](https://stackoverflow.com/questions/49450503/copy-content-from-popup/49487767#49487767) I think we probably need a value of `text` as `auto` will resolve to `none` if a parent has `user-select: none` as is the case in OpenLayers due to `user-select: none` being set on `.ol-viewport`.

Further details of `auto` in spec: https://drafts.csswg.org/css-ui-4/#valdef-user-select-auto

Reply:
@walkermatt feel free to open a pull request to change from `auto` to `text`
Note: the original PR #6729 had `text` instead of `auto`; I don't remember why it was changed.